lib/execute.js

```diff
@@ -97,7 +97,8 @@ const createTemporaryScriptFile = (extension, contents, callback) => {
  *   console.log(output);
  * });
  */
-const executeScript = (scriptPath, callback, times = 5) => {
+const executeScript = (scriptPath, callback, times) => {
+  times = times || 5;
```

Comment:
Turns out that this is why the tests are failing (the failure on master was due to timeout); when `times` is zero, it sets it back to `5`:

```
given EAGAIN errors when running the script
  drivelist write /var/folders/_m/xht9586n6791b9p_c8x4k53m0000gn/T/drivelist-8d954901e447: OK +508ms
  drivelist Got EAGAIN, retrying... 5 +1ms
  drivelist Got EAGAIN, retrying... 4 +506ms
  drivelist Got EAGAIN, retrying... 3 +504ms
  drivelist Got EAGAIN, retrying... 2 +506ms
  drivelist Got EAGAIN, retrying... 1 +504ms
  drivelist Got EAGAIN, retrying... 5 +501ms
  drivelist Got EAGAIN, retrying... 4 +504ms
```

As we want `times = 0` to disable retries, this would need to be
```js
times = times === undefined ? 5 : times;
```